README.md

```diff
@@ -1 +1,81 @@
-# public-R-bedr
+# bedr
+
+1. [Description](#description)
+2. [Installation](#installation)
+3. [Resources](#resources)
+4. [Getting help](#getting-help)
+5. [Citation information](#citation-information)
+6. [License](#license)
+
+## Description
+
+Genomic regions processing using open-source command line tools such as `BEDTools`, `BEDOPS` and `Tabix`.
+These tools offer scalable and efficient utilities to perform genome arithmetic e.g indexing, formatting and merging.
+bedr API enhances access to these tools as well as offers additional utilities for genomic regions processing.
+
+## Installation
+
+To install the latest public release of bedr from CRAN:
+
+```R
+install.packages('bedr');
+```
+
+Or to install the latest development version from Github:
+
+```R
+# If `devtools` is not already installed, run:
+# install.packages("devtools");
+
+devtools::install_github('uclahs-cds/public-R-bedr');
+```
+
+## Resources
+
+Available resources for bedr include the package [CRAN page](https://cran.r-project.org/web/packages/bedr/index.html), [reference manual](https://cran.r-project.org/web/packages/bedr/index.html) and [vignette](https://cran.r-project.org/web/packages/bedr/vignettes/Using-bedr.html).
+
+## Getting help
+
+For guidance or support with bedr check out [Discussions](https://github.com/uclahs-cds/public-R-bedr/discussions)
+
+See [Issues](https://github.com/uclahs-cds/public-R-bedr/issues) to submit bugs, suggest new features or view current works
+
+[Pull requests](https://github.com/uclahs-cds/public-R-bedr/pulls) are also open for discussion.
+
+## Citation information
+
+To cite package `bedr` in publications, use:
+
+Haider, S., Waggott, D., Lalonde, E. et al. A bedr way of genomic interval processing. _Source Code Biol Med_ **11**, 14 (2016). https://doi.org/10.1186/s13029-016-0059-5
+
+A BibTeX entry for LaTeX users is:
+
+```BibTeX
+@Article{,
+    title = {A bedr way of genomic interval processing},
+    journal = {Source Code for Biology and Medicine},
+    doi = {10.1186/s13029-016-0059-5},
+    url = {https://doi.org/10.1186/s13029-016-0059-5},
+    volume = {11},
+    number = {14},
+    year = {2016},
+    month = {December},
+    day = {15},
+    issn = {1751-0473},
+    author = {Syed Haider and Daryl Waggott and Emilie Lalonde and Clement Fung and Fei-Fei Liu and Paul C. Boutros},
+  }
+```
+
+## License
+
+Author: Syed Haider, Daryl Waggott, Emilie Lalonde, Clement Fung & Paul C. Boutros (PBoutros@mednet.ucla.edu)
```

Comment:
Wondering if I should add repo contributors here, like Helena or Nick to this section. This hasn't been super consistent, I added contributors for BPG but not for VennDiagram. If yes, I will go and add them for VennDiagram as well.

Reply:
Good idea, yes please!

Reply:
It would be nice to add the contributors section using GitHub contributors like BPG, which is automatically managed. For the license Author section, I think we can only include the internal contributors.